app/src/main/res/layout/view_image_title_description.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/imageTitleDescParentLayout"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:layout_marginTop="@dimen/suggested_edits_image_title_view_top_margin"
+    android:baselineAligned="false"
+    android:orientation="horizontal">
+
+    <FrameLayout
+        android:id="@+id/imageLayout"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginBottom="16dp">
+
+        <org.wikipedia.views.CircularProgressBar
```

Comment:
@cooltey 
>Looks like the edit quality icon size is much bigger than the other one.

Reply:
not yet..

Reply:
I have fixed this in the latest commit... but i guess the original image has a bit of padding in it. Will ask Robin for a correct one.